solr/core/src/java/org/apache/solr/core/SolrCore.java

```diff
@@ -1129,7 +1129,7 @@ protected SolrCore(
       initWriters();
       qParserPlugins.init(QParserPlugin.standardPlugins, this);
       valueSourceParsers.init(ValueSourceParser.standardValueSourceParsers, this);
-      transformerFactories.init(TransformerFactory.defaultFactories, this);
+      transformerFactories.init(TransformerFactory.builtIns(coreContainer.getConfig()), this);
```

Comment:
I like the name "builtIns" best so I chose it.  "implicits" would be my second choice.  I like a general name, thus not including "factories" as some places where we would use this pattern are not even "factories".  "default" isn't terrible but less good.